### ISSUE TITLE
How to run it on Windows?

### ISSUE THREAD

**Issue by seiferflo** [nikdoof/NextAction#3]
Hi, Sorry to open a new issue for this, but can anyone tell me how to run that script on Windows ?

I've tried: C:\Python27>python nextaction.py -a APIKEY
Traceback (most recent call last):
  File "nextaction.py", line 7, in <module>
    from todoist.api import TodoistAPI
ImportError: No module named todoist.api

In the script, where should I put the address of the sync api https://todoist.com/API/v6/sync & my token please?
Thanks in advance for your feedback.

**Comment by nikdoof** [comment]
I should really update the installation documents...

It should be simple, run ```python setup.py install``` and it should install all the dependencies and scripts that are required. On a Windows installation you should have a script called ```C:\Python27\Scripts\nextaction``` to run the tool itself.

In regards to running the script itself, you just replace ```APIKEY``` with your Token, the Sync API URL is took care of by the application.

**Comment by seiferflo** [comment]
Great thanks for your feedback. I think I've managed to make it work on one wheel but the behavior doesn't seem to be correct. 

A few notes to add to your installation documents from a noob perspective (correct me if I'm wrong):
- Download python v2.7 and install it with `standard option` (I made the mistake of choosing custom so I didn't install the "setuptools" module...
- What's `todoist-python` package and how to install it?
- Download the latest source and paste it in which folder ...?
- Run the following command using cmd `python setup.py install` 
- In your comment you say to run the script from the folder `C:\Python27\Scripts\nextaction` but there is no `nextaction` folder created. I see only a `nextaction.exe`, `nextaction.exe.manifest` & `nextaction-script.py`. 
Looking at your source code file, there is a `nextaction.py` of 8KB when the `nextaction-script.py` is only 1KB. It's unclear at this stage which one to use... 
- Create a `@next_action` label in Todoist
- Run the following command: `python nextaction.py -a <API Key>` (without the <> The API key can be found in `Todoist Settings`, `Account`)

From that point in todoist, I've created this project: https://i.imgur.com/KWBd0Nl.jpg
- if I add an `_` https://i.imgur.com/u3a9urA.jpg all sub-tasks are marked as next_action which I don't understand why as I would have thought only Sub-Task A1 would be the next-action
- If I remove the `_` https://i.imgur.com/jy28buD.jpg the `@next_action` labels do not disappear at all. Only way to make it go is to remove it manually.
- If I add the `_` to a task only, nothing happens https://i.imgur.com/ro8LhpS.jpg
- If I add an `.` to the project, again all sub-tasks are marked as next-action https://i.imgur.com/1KNWgBh.jpg
- If I add a `.` added to a task to run them in parallel, nothing happens https://i.imgur.com/2czosQX.jpg
- Each refresh takes about a minute, and sometimes I need to press the sync button, which doesn't seem normal.

Overall, I think I didn't install it properly so I'm awaiting for your feedback on the above procedure. Many thanks in advance.

**Comment by seiferflo** [comment]
Hi nikdoof. 
Funny that you just closed this request. I was there last night trying to figure out again how to set it up again... 
From my understanding the script just doesn't work properly with the new API , correct? 
If not, it would be lovely if you could make a tiny tutorial on how to install it on Windows because it is not as easy as it seems. 
Best Regards
